tests/Http/TagsTest.php

```diff
@@ -22,18 +21,18 @@ public function testAddingATagToAPost()
         $post = factory(Post::class)->create();
 
         // Apply the tag to the post
-        $this->actingAsAdmin()->post('tags', [
-                'post_id' => $post->id,
-                'tag_name' => 'Good Photo',
-            ]);
+        $response = $this->actingAsAdmin()->post('tags', [
+            'post_id' => $post->id,
+            'tag_name' => 'Good Photo',
+        ]);
 
-        $this->assertResponseStatus(200);
+        $response->assertSuccessful();
```

Comment:
this is so cool @DFurnes ! When should we use this vs `$response->assertStatus(200);`?

Reply:
It's subjective I think. This will accept any non-error code which might be nice for those cases where we used to return the wrong `201` or `200` status codes.

Reply:
cool!